tests/gold_tests/dns/dns_host_down.test.py

```diff
@@ -22,6 +22,10 @@
 Test.Summary = '''
 Verify ATS handles down origin servers with cached domain correctly.
 '''
+# This test is dependent on the new HostDB restructure that is not in 9.2.x.
+# The code is still applicable for 9.2.x but the autest created to show its
+# functionality is a workaround that only applies to 10-Dev.
```

Comment:
I suggest a few tweaks to the second sentence:

* `The code` -> `The production patch associated with this test`
* `is a workaround` -> `relies upon the restructured HostDB's relationship with down nameservers`

Reply:
Thank you!